fix(components): [tabs] fix: tab remove keydown event handling

tuskermanshu opened the pull request:
This fix prevents accidental tab closures when interacting with child elements

closed #18045

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

tuskermanshu committed:
fix(components): [tabs] fix: tab remove keydown event handling

This fix prevents accidental tab closures when interacting with child elements

closed #18045

dadaguai-git commented:
/label Components:[tabs]

dadaguai-git commented:
[Playground ](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtdGFicyB2LW1vZGVsPVwiZWRpdGFibGVUYWJzVmFsdWVcIiB0eXBlPVwiY2FyZFwiIGVkaXRhYmxlIGNsYXNzPVwiZGVtby10YWJzXCIgQGVkaXQ9XCJoYW5kbGVUYWJzRWRpdFwiPlxuICAgIDxlbC10YWItcGFuZSB2LWZvcj1cIml0ZW0gaW4gZWRpdGFibGVUYWJzXCIgOmtleT1cIml0ZW0ubmFtZVwiIDpsYWJlbD1cIml0ZW0udGl0bGVcIiA6bmFtZT1cIml0ZW0ubmFtZVwiPlxuICAgICAgPHRlbXBsYXRlICNsYWJlbD5cbiAgICAgICAgPGVsLWlucHV0IHYtbW9kZWw9XCJpdGVtLmNvbnRlbnRcIiBAaW5wdXQ9XCJoYW5kbGVJbnB1dFwiIEBjbGNpay5zdG9wLnByZXZlbnQgQGtleWRvd249XCJoYW5kbGVLZXlEb3duXCIvPlxuICAgICAgPC90ZW1wbGF0ZT5cbiAgICA8L2VsLXRhYi1wYW5lPlxuICA8L2VsLXRhYnM+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHR5cGUgeyBUYWJQYW5lTmFtZSB9IGZyb20gJ2VsZW1lbnQtcGx1cydcblxubGV0IHRhYkluZGV4ID0gMlxuY29uc3QgZWRpdGFibGVUYWJzVmFsdWUgPSByZWYoJzInKVxuY29uc3QgZWRpdGFibGVUYWJzID0gcmVmKFtcbiAge1xuICAgIHRpdGxlOiAnVGFiIDEnLFxuICAgIG5hbWU6ICcxJyxcbiAgICBjb250ZW50OiAnVGFiIDEgY29udGVudCcsXG4gIH0sXG4gIHtcbiAgICB0aXRsZTogJ1RhYiAyJyxcbiAgICBuYW1lOiAnMicsXG4gICAgY29udGVudDogJ1RhYiAyIGNvbnRlbnQnLFxuICB9LFxuXSlcblxuY29uc3QgaGFuZGxlSW5wdXQgPSAoZXZlbnQpID0+IHtcbiAgY29uc29sZS5sb2coXCJldmVudFwiLGV2ZW50KVxuICAvLyDlpITnkIbovpPlhaXmoYbnmoTlgLzlj5jljJZcbn07XG5cbmNvbnN0IGhhbmRsZUtleURvd24gPSAoZSkgPT57XG4gIGUuc3RvcFByb3BhZ2F0aW9uKClcbn1cblxuY29uc3QgaGFuZGxlVGFic0VkaXQgPSAoXG4gIHRhcmdldE5hbWU6IFRhYlBhbmVOYW1lIHwgdW5kZWZpbmVkLFxuICBhY3Rpb246ICdyZW1vdmUnIHwgJ2FkZCdcbikgPT4ge1xuICBjb25zb2xlLmxvZyh0YXJnZXROYW1lLCBhY3Rpb24pXG4gIGlmIChhY3Rpb24gPT09ICdhZGQnKSB7XG4gICAgY29uc3QgbmV3VGFiTmFtZSA9IGAkeysrdGFiSW5kZXh9YFxuICAgIGVkaXRhYmxlVGFicy52YWx1ZS5wdXNoKHtcbiAgICAgIHRpdGxlOiAnTmV3IFRhYicsXG4gICAgICBuYW1lOiBuZXdUYWJOYW1lLFxuICAgICAgY29udGVudDogJ05ldyBUYWIgY29udGVudCcsXG4gICAgfSlcbiAgICBlZGl0YWJsZVRhYnNWYWx1ZS52YWx1ZSA9IG5ld1RhYk5hbWVcbiAgfSBlbHNlIGlmIChhY3Rpb24gPT09ICdyZW1vdmUnKSB7XG4gICAgY29uc3QgdGFicyA9IGVkaXRhYmxlVGFicy52YWx1ZVxuICAgIGxldCBhY3RpdmVOYW1lID0gZWRpdGFibGVUYWJzVmFsdWUudmFsdWVcbiAgICBpZiAoYWN0aXZlTmFtZSA9PT0gdGFyZ2V0TmFtZSkge1xuICAgICAgdGFicy5mb3JFYWNoKCh0YWIsIGluZGV4KSA9PiB7XG4gICAgICAgIGlmICh0YWIubmFtZSA9PT0gdGFyZ2V0TmFtZSkge1xuICAgICAgICAgIGNvbnN0IG5leHRUYWIgPSB0YWJzW2luZGV4ICsgMV0gfHwgdGFic1tpbmRleCAtIDFdXG4gICAgICAgICAgaWYgKG5leHRUYWIpIHtcbiAgICAgICAgICAgIGFjdGl2ZU5hbWUgPSBuZXh0VGFiLm5hbWVcbiAgICAgICAgICB9XG4gICAgICAgIH1cbiAgICAgIH0pXG4gICAgfVxuXG4gICAgZWRpdGFibGVUYWJzVmFsdWUudmFsdWUgPSBhY3RpdmVOYW1lXG4gICAgZWRpdGFibGVUYWJzLnZhbHVlID0gdGFicy5maWx0ZXIoKHRhYikgPT4gdGFiLm5hbWUgIT09IHRhcmdldE5hbWUpXG4gIH1cbn1cbjwvc2NyaXB0PlxuXG48c3R5bGU+XG4uZGVtby10YWJzPi5lbC10YWJzX19jb250ZW50IHtcbiAgcGFkZGluZzogMzJweDtcbiAgY29sb3I6ICM2Yjc3OGM7XG4gIGZvbnQtc2l6ZTogMzJweDtcbiAgZm9udC13ZWlnaHQ6IDYwMDtcbn1cbjwvc3R5bGU+XG4iLCJlbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgeyBnZXRDdXJyZW50SW5zdGFuY2UgfSBmcm9tICd2dWUnXG5pbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgY29uc3Qgc3R5bGVzID0gWydodHRwczovL3ByZXZpZXctMTgwNDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnLCAnaHR0cHM6Ly9wcmV2aWV3LTE4MDQ2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJ10ubWFwKChzdHlsZSkgPT4ge1xuICAgIHJldHVybiBuZXcgUHJvbWlzZSgocmVzb2x2ZSwgcmVqZWN0KSA9PiB7XG4gICAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgICAgbGluay5ocmVmID0gc3R5bGVcbiAgICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgICB9KVxuICB9KVxuICByZXR1cm4gUHJvbWlzZS5hbGxTZXR0bGVkKHN0eWxlcylcbn1cbiIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcInR5cGVzXCI6IFtcImVsZW1lbnQtcGx1cy9nbG9iYWwuZC50c1wiXSxcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWUsXG4gICAgXCJhbGxvd0pzXCI6IHRydWUsXG4gICAgXCJjaGVja0pzXCI6IHRydWVcbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJQbGF5Z3JvdW5kTWFpbi52dWUiOiI8c2NyaXB0IHNldHVwPlxuaW1wb3J0IEFwcCBmcm9tICcuL0FwcC52dWUnXG5pbXBvcnQgeyBzZXR1cEVsZW1lbnRQbHVzIH0gZnJvbSAnLi9lbGVtZW50LXBsdXMuanMnXG5zZXR1cEVsZW1lbnRQbHVzKClcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxBcHAgLz5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJ2dWVcIjogXCJodHRwczovL3VucGtnLmNvbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vdW5wa2cuY29tL0B2dWUvc2hhcmVkQGxhdGVzdC9kaXN0L3NoYXJlZC5lc20tYnVuZGxlci5qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9wcmV2aWV3LTE4MDQ2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIixcbiAgICBcIkBlbGVtZW50LXBsdXMvaWNvbnMtdnVlXCI6IFwiaHR0cHM6Ly91bnBrZy5jb20vQGVsZW1lbnQtcGx1cy9pY29ucy12dWVAMi9kaXN0L2luZGV4Lm1pbi5qc1wiXG4gIH0sXG4gIFwic2NvcGVzXCI6IHt9XG59IiwiX28iOnsic2hvd0hpZGRlbiI6dHJ1ZSwic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctMTgwNDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MifX0=)

tuskermanshu committed:
fix(components): [tabs] refactor:use tab ID for precise keydown handling

This fix prevents accidental tab closures when interacting with child elements

closed #18045

btea commented:
Thank you for your contribution, but I think this modification seems not perfect enough. If the user uses a custom slot, perhaps he should follow the example provided in [my comment](https://github.com/element-plus/element-plus/issues/18045#issuecomment-2315380631) to prevent event bubbling by himself.

Here is a scenario I thought of, which may cause keyboard deletion to fail according to your modification. [example](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtdGFicyB2LW1vZGVsPVwiZWRpdGFibGVUYWJzVmFsdWVcIiB0eXBlPVwiY2FyZFwiIGVkaXRhYmxlIGNsYXNzPVwiZGVtby10YWJzXCIgQGVkaXQ9XCJoYW5kbGVUYWJzRWRpdFwiPlxuICAgIDxlbC10YWItcGFuZSB2LWZvcj1cIml0ZW0gaW4gZWRpdGFibGVUYWJzXCIgOmtleT1cIml0ZW0ubmFtZVwiIDpsYWJlbD1cIml0ZW0udGl0bGVcIiA6bmFtZT1cIml0ZW0ubmFtZVwiPlxuICAgICAgPHRlbXBsYXRlICNsYWJlbD5cbiAgICAgICAgPGRpdiB0YWJpbmRleD1cIi0xXCI+e3sgaXRlbS50aXRsZSB9fTwvZGl2PlxuICAgICAgPC90ZW1wbGF0ZT5cbiAgICA8L2VsLXRhYi1wYW5lPlxuICA8L2VsLXRhYnM+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHR5cGUgeyBUYWJQYW5lTmFtZSB9IGZyb20gJ2VsZW1lbnQtcGx1cydcblxubGV0IHRhYkluZGV4ID0gMlxuY29uc3QgZWRpdGFibGVUYWJzVmFsdWUgPSByZWYoJzInKVxuY29uc3QgZWRpdGFibGVUYWJzID0gcmVmKFtcbiAge1xuICAgIHRpdGxlOiAnVGFiIDEnLFxuICAgIG5hbWU6ICcxJyxcbiAgICBjb250ZW50OiAnVGFiIDEgY29udGVudCcsXG4gIH0sXG4gIHtcbiAgICB0aXRsZTogJ1RhYiAyJyxcbiAgICBuYW1lOiAnMicsXG4gICAgY29udGVudDogJ1RhYiAyIGNvbnRlbnQnLFxuICB9LFxuXSlcblxuY29uc3QgaGFuZGxlSW5wdXQgPSAoZXZlbnQpID0+IHtcbiAgY29uc29sZS5sb2coXCJldmVudFwiLGV2ZW50KVxuICAvLyDlpITnkIbovpPlhaXmoYbnmoTlgLzlj5jljJZcbn07XG5cbmNvbnN0IGhhbmRsZUtleURvd24gPSAoZSkgPT57XG4gIGUuc3RvcFByb3BhZ2F0aW9uKClcbn1cblxuY29uc3QgaGFuZGxlVGFic0VkaXQgPSAoXG4gIHRhcmdldE5hbWU6IFRhYlBhbmVOYW1lIHwgdW5kZWZpbmVkLFxuICBhY3Rpb246ICdyZW1vdmUnIHwgJ2FkZCdcbikgPT4ge1xuICBjb25zb2xlLmxvZyh0YXJnZXROYW1lLCBhY3Rpb24pXG4gIGlmIChhY3Rpb24gPT09ICdhZGQnKSB7XG4gICAgY29uc3QgbmV3VGFiTmFtZSA9IGAkeysrdGFiSW5kZXh9YFxuICAgIGVkaXRhYmxlVGFicy52YWx1ZS5wdXNoKHtcbiAgICAgIHRpdGxlOiAnTmV3IFRhYicsXG4gICAgICBuYW1lOiBuZXdUYWJOYW1lLFxuICAgICAgY29udGVudDogJ05ldyBUYWIgY29udGVudCcsXG4gICAgfSlcbiAgICBlZGl0YWJsZVRhYnNWYWx1ZS52YWx1ZSA9IG5ld1RhYk5hbWVcbiAgfSBlbHNlIGlmIChhY3Rpb24gPT09ICdyZW1vdmUnKSB7XG4gICAgY29uc3QgdGFicyA9IGVkaXRhYmxlVGFicy52YWx1ZVxuICAgIGxldCBhY3RpdmVOYW1lID0gZWRpdGFibGVUYWJzVmFsdWUudmFsdWVcbiAgICBpZiAoYWN0aXZlTmFtZSA9PT0gdGFyZ2V0TmFtZSkge1xuICAgICAgdGFicy5mb3JFYWNoKCh0YWIsIGluZGV4KSA9PiB7XG4gICAgICAgIGlmICh0YWIubmFtZSA9PT0gdGFyZ2V0TmFtZSkge1xuICAgICAgICAgIGNvbnN0IG5leHRUYWIgPSB0YWJzW2luZGV4ICsgMV0gfHwgdGFic1tpbmRleCAtIDFdXG4gICAgICAgICAgaWYgKG5leHRUYWIpIHtcbiAgICAgICAgICAgIGFjdGl2ZU5hbWUgPSBuZXh0VGFiLm5hbWVcbiAgICAgICAgICB9XG4gICAgICAgIH1cbiAgICAgIH0pXG4gICAgfVxuXG4gICAgZWRpdGFibGVUYWJzVmFsdWUudmFsdWUgPSBhY3RpdmVOYW1lXG4gICAgZWRpdGFibGVUYWJzLnZhbHVlID0gdGFicy5maWx0ZXIoKHRhYikgPT4gdGFiLm5hbWUgIT09IHRhcmdldE5hbWUpXG4gIH1cbn1cbjwvc2NyaXB0PlxuXG48c3R5bGU+XG4uZGVtby10YWJzPi5lbC10YWJzX19jb250ZW50IHtcbiAgcGFkZGluZzogMzJweDtcbiAgY29sb3I6ICM2Yjc3OGM7XG4gIGZvbnQtc2l6ZTogMzJweDtcbiAgZm9udC13ZWlnaHQ6IDYwMDtcbn1cbjwvc3R5bGU+XG4iLCJlbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgeyBnZXRDdXJyZW50SW5zdGFuY2UgfSBmcm9tICd2dWUnXG5pbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgY29uc3Qgc3R5bGVzID0gWydodHRwczovL3ByZXZpZXctMTgwNDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnLCAnaHR0cHM6Ly9wcmV2aWV3LTE4MDQ2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJ10ubWFwKChzdHlsZSkgPT4ge1xuICAgIHJldHVybiBuZXcgUHJvbWlzZSgocmVzb2x2ZSwgcmVqZWN0KSA9PiB7XG4gICAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgICAgbGluay5ocmVmID0gc3R5bGVcbiAgICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgICB9KVxuICB9KVxuICByZXR1cm4gUHJvbWlzZS5hbGxTZXR0bGVkKHN0eWxlcylcbn1cbiIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcInR5cGVzXCI6IFtcImVsZW1lbnQtcGx1cy9nbG9iYWwuZC50c1wiXSxcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWUsXG4gICAgXCJhbGxvd0pzXCI6IHRydWUsXG4gICAgXCJjaGVja0pzXCI6IHRydWVcbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJQbGF5Z3JvdW5kTWFpbi52dWUiOiI8c2NyaXB0IHNldHVwPlxuaW1wb3J0IEFwcCBmcm9tICcuL0FwcC52dWUnXG5pbXBvcnQgeyBzZXR1cEVsZW1lbnRQbHVzIH0gZnJvbSAnLi9lbGVtZW50LXBsdXMuanMnXG5zZXR1cEVsZW1lbnRQbHVzKClcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxBcHAgLz5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJ2dWVcIjogXCJodHRwczovL3VucGtnLmNvbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vdW5wa2cuY29tL0B2dWUvc2hhcmVkQGxhdGVzdC9kaXN0L3NoYXJlZC5lc20tYnVuZGxlci5qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9wcmV2aWV3LTE4MDQ2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIixcbiAgICBcIkBlbGVtZW50LXBsdXMvaWNvbnMtdnVlXCI6IFwiaHR0cHM6Ly91bnBrZy5jb20vQGVsZW1lbnQtcGx1cy9pY29ucy12dWVAMi9kaXN0L2luZGV4Lm1pbi5qc1wiXG4gIH0sXG4gIFwic2NvcGVzXCI6IHt9XG59IiwiX28iOnsic2hvd0hpZGRlbiI6dHJ1ZSwic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctMTgwNDYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MifX0=)

btea commented:
Based on the above reasons, I will close this PR.
